libcni/api_test.go

```diff
@@ -122,11 +123,13 @@ func newPluginInfo(configValue, prevResult string, injectDebugFilePath bool, res
 }
 
 func resultCacheFilePath(cacheDirPath, netName string, rt *libcni.RuntimeConf) string {
-	return filepath.Join(cacheDirPath, "results", netName+"-"+rt.ContainerID+"-"+rt.IfName)
+	fName := fmt.Sprintf(netName + "-" + rt.ContainerID + "-" + rt.IfName)
```

Comment:
O_O
What is the reason for sum to be enclosed in `fmt.Sprintf`?

Reply:
@mccv1r0 Yeah, either fmt.Sprintf("%s-%s-%s", ...) or just netName + "-" .... Best to be consistent I think.